builtin/settingtypes.txt

```diff
@@ -1680,6 +1680,12 @@ language (Language) enum   ,be,ca,cs,da,de,en,eo,es,et,fr,he,hu,id,it,ja,jbo,ko,
 #    -    verbose
 debug_log_level (Debug log level) enum action ,none,error,warning,action,info,verbose
 
+#    ANSI colored logs: red error log, yellow warning and grey info and verbose logs
+#    "yes" always enables it,
+#    "tty" enables it when printing to terminal and
```

Comment:
https://askubuntu.com/questions/481906/what-does-tty-stand-for/481915#481915
l'll call it "detect" because automatic decision of whether it should add colour codes is desired.